src/components/App.js

```diff
@@ -246,26 +249,35 @@ class App extends Component {
 
         <div className="bottom-bar">
           <div className="bottom-left">
-            <div id="previous" onClick={this.previousTrack} title="Play Last">
-              <i className="fa fa-backward fa-2x" />
+            <div title="Play Previous">
+              <TiArrowLeftThick
+                onClick={this.previousTrack}
+                size={35}
```

Comment:
Since this value is reused for all the icons, would it make sense to move the value into `constants.js` instead?

Reply:
Yes, on it.